src/sql/workbench/parts/query/common/queryResultsInput.ts

```diff
@@ -43,19 +42,6 @@ export class ResultsViewState {
  */
 export class QueryResultsInput extends EditorInput {
 
-	// Tracks if the editor that holds this input should be visible (i.e. true if a query has been run)
-	private _visible: boolean;
-
-	// Tracks if the editor has holds this input has has bootstrapped angular yet
-	private _hasBootstrapped: boolean;
-
-	// Holds the HTML content for the editor when the editor discards this input and loads another
-	private _editorContainer: HTMLElement;
-	public css: HTMLStyleElement;
-
-	public readonly onRestoreViewStateEmitter = new Emitter<void>();
-	public readonly onSaveViewStateEmitter = new Emitter<void>();
```

Comment:
were these not being used or something?

Reply:
Nope, must have been dead code from when I initially did the refactor on query grid to move it away from angular.